src/homepageExperience/components/OptionAccordion/OptionAccordionElement.tsx

```diff
@@ -5,7 +5,7 @@ import React, {FC} from 'react'
 import {Accordion, Heading, HeadingElement} from '@influxdata/clockface'
 
 interface OwnProps {
-  elementTitle: string
+  elementTitle: string | JSX.Element
   elementDescription?: string
```

Comment:
LGTM: since `OptionAccordionElement` renders `{elementTitle}` and doesn't rely on it having a specific type, this works swell.